android/src/amazon/java/com/dooboolab/RNIap/RNIapAmazonModule.java

```diff
@@ -87,7 +87,6 @@ public void getPurchaseHistoryByType(final String type, final Promise promise) {
   public void buyItemByType(
       final String type,
       final String sku,
-      final String oldSku,
```

Comment:
Dropping this without updating the TypeScript call site (https://github.com/dooboolab/react-native-iap/blob/master/src/iap.ts#L320) leads to an immediate crash.